editors/code/package.json

```diff
@@ -323,14 +323,6 @@
             {
                 "title": "general",
                 "properties": {
-                    "rust-analyzer.cargoRunner": {
```

Comment:
Also removes this, the custom runner was broken for ages already so clearly no one is depending on it